virt/managing_vms/virt-managing-vms-openshift-pipelines.adoc

```diff
@@ -43,6 +44,28 @@ include::modules/virt-running-ssp-pipeline-web.adoc[leveloffset=+2]
 
 include::modules/virt-running-ssp-pipeline-cli.adoc[leveloffset=+2]
 
+[id="virt-deprecated-tasks_{context}"]
+== Removing deprecated or unused resources
+
+You can clean up Tekton-Tasks-Operator resources associated with the operator.
+
+.Procedure
+
+* To remove any remaining Tekton-related resources from the cluster, run the following command:
++
+[source,terminal]
+----
+oc delete clusterroles,rolebindings,serviceaccounts,configmaps,pipelines,tasks \
```

Comment:
```suggestion
$ oc delete clusterroles,rolebindings,serviceaccounts,configmaps,pipelines,tasks \
```
Code blocks should start with a `$`. See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference